src/main/resources/application.properties

```diff
@@ -390,3 +390,4 @@ report.property.file.allMdo=ContentCompetencyMapping.csv
 operational.reports.passwordlength=15
 operational.reports.unzip.password=123456
 es.default.result.limit=250
+secret.key.primaryEmail.validation=secretKey
```

Comment:
This is not primaryEmail validation... it's extPatch operation validation.
rename the variable to -> secretKeyTokenValidation

Reply:
1. changed to secret.key.secretKeyTokenValidation